`go build` shows linking errors

`go build` shows many lines of linking errors: 

```
...
...
1565 SimdSsse3Sobel.cpp:(.text.startup+0xc): undefined reference to `std::ios_base::Init::Init()'
1566 SimdSsse3Sobel.cpp:(.text.startup+0x13): undefined reference to `std::ios_base::Init::~Init()'
1567 /home/vadmeste/work/x/Simd/prj/cmake/libSimdSsse3.a(SimdSsse3Texture.cpp.o): In function `_GLOBAL__sub_I_SimdSsse3Texture.cpp':
1568 SimdSsse3Texture.cpp:(.text.startup+0xc): undefined reference to `std::ios_base::Init::Init()'
1569 SimdSsse3Texture.cpp:(.text.startup+0x13): undefined reference to `std::ios_base::Init::~Init()'
1570 /home/vadmeste/work/x/Simd/prj/cmake/libSimdSsse3.a(SimdSsse3BgrToYuv.cpp.o): In function `_GLOBAL__sub_I_SimdSsse3BgrToYuv.cpp':
1571 SimdSsse3BgrToYuv.cpp:(.text.startup+0xc): undefined reference to `std::ios_base::Init::Init()'
1572 SimdSsse3BgrToYuv.cpp:(.text.startup+0x13): undefined reference to `std::ios_base::Init::~Init()'
1573 /home/vadmeste/work/x/Simd/prj/cmake/libSimdSsse3.a(SimdSsse3AlphaBlending.cpp.o): In function `_GLOBAL__sub_I_SimdSsse3AlphaBlending.cpp':
1574 SimdSsse3AlphaBlending.cpp:(.text.startup+0xc): undefined reference to `std::ios_base::Init::Init()'
1575 SimdSsse3AlphaBlending.cpp:(.text.startup+0x13): undefined reference to `std::ios_base::Init::~Init()'
1576 /home/vadmeste/work/x/Simd/prj/cmake/libSimdSsse3.a(SimdSsse3YuvToBgr.cpp.o): In function `_GLOBAL__sub_I_SimdSsse3YuvToBgr.cpp':
1577 SimdSsse3YuvToBgr.cpp:(.text.startup+0xc): undefined reference to `std::ios_base::Init::Init()'
1578 SimdSsse3YuvToBgr.cpp:(.text.startup+0x13): undefined reference to `std::ios_base::Init::~Init()'
1579 collect2: error: ld returned 1 exit status
```

It seems `#cgo LDFLAGS: -lstdc++` is not working properly as expected in my machine. However, the problem is resolved while creating an empty dummy.cpp file as encouraged here https://github.com/golang/go/issues/18460

--- a/dummy.cpp
+++ b/dummy.cpp
@@ -1,0 +1,4 @@
+
+// This is a dummy C++ file which enforces cgo to link
+// against C++ library. It can be removed if we ever 
+// write some C++ code later.

